Analysis/Tasks/PWGHF/HFLcK0spCandidateSelector.cxx

```diff
@@ -0,0 +1,336 @@
+      LOG(DEBUG) << "v0 pt cut failed, from cascade --> " << hfCandCascade.ptProng0() << ", cut --> " << cuts[pTBin][3];
+      return false; // pt of the K0
+    }
+
+    if (hfCandCascade.ptProng1() < cuts[pTBin][4]) {
+      LOG(DEBUG) << "bach pt cut failed, from cascade --> " << hfCandCascade.ptProng1() << " , cut --> " << cuts[pTBin][4];
+      return false; // pt of the p
+    }
+
+    if (hfCandCascade.pt() < cuts[pTBin][5]) {
+      LOG(DEBUG) << "cand pt cut failed, from cascade --> " << hfCandCascade.pt() << ", cut --> " << cuts[pTBin][5];
+      return false; // pt of the Lc
+    }
+
+    if (std::abs(hfCandCascade.impactParameter1()) > cuts[pTBin][6]) {
+      LOG(DEBUG) << "d0 bach cut failed, in cascade --> " << hfCandCascade.impactParameter1() << ", cut --> " << cuts[pTBin][6];
+      return false; // d0 of the bachelor
+    }
+
+    /*
+    if ((std::abs(hfCandCascade.dcapostopv()) > cuts[pTBin][7]) || (std::abs(hfCandCascade.dcanegtopv()) > cuts[pTBin][7])) {
+      LOG(DEBUG) << "v0 daugh cut failed, positive v0 daugh --> " << hfCandCascade.dcapostopv() << ", negative v0 daugh --> " << hfCandCascade.dcanegtopv() << " , cut --> " << cuts[pTBin][7];
+      return false; // d0 of the K0s daughters
+    }
+    */
+
+    if (std::abs(hfCandCascade.impactParameter0()) > cuts[pTBin][6]) {
+      LOG(DEBUG) << "d0 v0 cut failed, in cascade --> " << hfCandCascade.impactParameter0() << ", cut --> " << cuts[pTBin][7];
+      return false; // d0 of the v0
+    }
+
+    return true;
+  }
+
+  /// Check if track is ok for TPC PID
+  /// \param track is the track
+  /// \note function to be expanded
+  /// \return true if track is ok for TPC PID
+  template <typename T>
+  bool validTPCPID(const T& track)
+  {
+    if (TMath::Abs(track.pt()) < d_pidTPCMinpT || TMath::Abs(track.pt()) >= d_pidTPCMaxpT) {
+      return false;
+    }
+    //if (track.TPCNClsFindable() < d_TPCNClsFindablePIDCut) return false;
+    return true;
+  }
+
+  /// Check if track is ok for TOF PID
+  /// \param track is the track
+  /// \note function to be expanded
+  /// \return true if track is ok for TOF PID
+  template <typename T>
+  bool validCombPID(const T& track)
+  {
+    if (TMath::Abs(track.pt()) > d_pidCombMaxp) { // is the pt sign used for the charge? If it is always positive, we should remove the abs
+      return false;
+    }
+    return true;
+  }
+
+  /// Check if track is compatible with given TPC Nsigma cut for a given flavour hypothesis
+  /// \param track is the track
+  /// \param nPDG is the flavour hypothesis PDG number
+  /// \param nSigmaCut is the nsigma threshold to test against
+  /// \note nPDG=211 pion  nPDG=321 kaon
+  /// \return true if track satisfies TPC PID hypothesis for given Nsigma cut
+  template <typename T>
+  bool selectionPIDTPC(const T& track, double nSigmaCut)
+  {
+    double nSigma = 100.0; //arbitarily large value
+    nSigma = track.tpcNSigmaPr();
+    return nSigma > nSigmaCut;
```

Comment:
`nSigma < nSigmaCut`
@nzardosh Shouldn't here be `std::abs(nSigma)`?

Reply:
I will add it, because I agree